packages/generator/src/v2.0/UNSTABLE_wireInterfaceTypeV2ToSdkObjectConst.ts

```diff
@@ -175,56 +175,25 @@ ${
 
       ${createObjectSet(interfaceDef, ids)}
 
-      ${createDefinition(interfaceDef, ontology, "Definition", ids)}
-
       ${createOsdkObject(interfaceDef, "OsdkObject", ids)}
       
     }    
 
-
-
-
-  /** @deprecated use ${interfaceDef.shortApiName}.Definition **/
-  export type ${objectDefIdentifier} = ${interfaceDef.shortApiName}.Definition;
-
-
-
+    ${createDefinition(interfaceDef, ontology, interfaceDef.shortApiName, ids)}
 
 `;
   }
 
   // FIXME: We need to fill in the imports
   // if we want links to work
   const imports: string[] = [];
-
+  definition;
   return `${imports.join("\n")}
     ${v2 ? getV2Types() : ""}
 
-    export const ${interfaceDef.shortApiName}: ${interfaceDef.shortApiName}.Definition = {
+    export const ${interfaceDef.shortApiName}: ${interfaceDef.shortApiName} = {
+      type: "interface",
+      apiName: "${interfaceDef.fullApiName}",
```

Comment:
We do type: "${object instanceof EnhancedObjectType ? "object" : "interface"}" in object. Is this needed here if no errors?

Reply:
If we do it there and its just for objects then its probably a mistake where I thought those lines were being reused

Reply:
Cool will fix in flup